python/chronos/use-case/pytorch-forecasting/DeepAR/deepar.py

```diff
@@ -0,0 +1,107 @@
+from pathlib import Path
```

Comment:
same problem as in #4348, please add
1. bigdl license
2. pytorch-forecasting license
3. adapted from ...